svmbir_test.py

```diff
@@ -0,0 +1,99 @@
+import matplotlib.pyplot as plt
```

Comment:
This file should be deleted before merge. It is just for testing purposes.

Reply:
Perhaps best to mark the PR as a draft until this file has been deleted.

Reply:
Is this still needed? It looks as if the PR is almost ready to go, so perhaps it should be deleted?

Reply:
Currently still using it. (Can remove it from the repo though if you prefer)

Reply:
No, we should only delete it once it's no longer needed.

Reply:
Done